tuf_conformance/client_runner.py

```diff
@@ -110,7 +112,16 @@ def _files_exist(self, roles: Iterable[str]) -> bool:
         local_metadata_files = sorted(os.listdir(self.metadata_dir))
         return all(x in local_metadata_files for x in expected_files)
 
-    def _content(self, role: str) -> bytes:
-        """Return role metadata as bytes"""
-        with open(os.path.join(self.metadata_dir, f"{role}.json"), "rb") as f:
-            return f.read()
+    def trusted_roles(self) -> dict[str, int]:
```

Comment:
I suppose this return value  could be `list[tuple(str, int)]` and the list could be sorted by mtime to match what `RepositorySimulator.metadata_statistics() `returns